pkgs/applications/virtualization/OVMF/default.nix

```diff
@@ -17,9 +17,13 @@ let
     throw "Unsupported architecture";
 
   version = lib.getVersion edk2;
+  buildType = if stdenv.isDarwin then
```

Comment:
```suggestion
  buildType = if stdenv.cc.isClang then
```

Reply:
I removed this argument as suggested below.